src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -201,7 +202,7 @@ class Selection extends React.Component<Props> {
         resourceStorePropertiesToRequest.forEach((propertyToRequest) => {
             const {name: parameterName, value: propertyName} = propertyToRequest;
             const propertyPath = typeof propertyName === 'string' ? propertyName : parameterName;
-            requestOptions[parameterName] = formInspector.getValueByPath('/' + propertyPath);
+            requestOptions[parameterName] = toJS(formInspector.getValueByPath('/' + propertyPath));
```

Comment:
Why was that necessary?

Reply:
because the value of the property might be an observable array and the `fast-deep-equal` package unfortunately does not compare the values inside of these 🙈 

for example, we need to do the same thing in the `MultiSelection` component: https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiSelection/MultiSelection.js#L56